enum_set.hh

```diff
@@ -239,6 +239,19 @@ public:
         _mask |= mask_for(e);
     }
 
+    template<enum_type e>
+    void toggle() {
+        const auto new_value = ~(_mask & mask_for<e>()) & mask_for<e>();
+        remove<e>();
+        _mask |= new_value;
+    }
+
+    void toggle(enum_type e) {
+        const auto new_value = ~(_mask & mask_for(e)) & mask_for(e);
```

Comment:
Isn't it enough to  `_mask ^= mask_for(e);` ?

Reply:
Indeed it is, v2 pushed.